src/pages/ProductPage.jsx

```diff
@@ -2,10 +2,10 @@ import ReservationCard from "../components/ReservationCard/ReservationCard";
 import FavoriteStay from "../components/FavoriteStay/FavoriteStay";
 import HostSummary from "../components/HostSummary/HostSummary";
 import hostImage from "../assets/images/host-raus.webp";
-import MapView from "../components/MapView/MapView";
+import MapView from "../components/mapView/MapView";
 import mapViewSampleImg from "./../assets/map-view-sample.png";
-import ProductGallery from "../components/ProductGallery/ProductGallery";
-import PersonProfile from "../components/PersonProfile/PersonProfile";
+import ProductGallery from "../components/productGallery/ProductGallery";
+import PersonProfile from "../components/personProfile/PersonProfile";
```

Comment:
should be the old version, on main the components starting with capital letters

Reply:
When I downloaded a ripo from the beginning, it complained of the wrong path, so I just solved it to work on it.

Reply:
I think in this file we can see the changes before they were solved. And I solved them already. Don't you think so?